cypress/features/regression/buttonIconPart1.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Button component
+  I want to check Button component properties
+
+  Background: Open Button component page default
+    Given I open "Button" component page
+
+  @positive
+  Scenario Outline: Set Button icon to <iconType>
+    When I check has icon checkbox
```

Comment:
Could we move `I check has icon checkbox` to Background?

Reply:
fix